bcgov/HOUS-permit-portal
# Feature/bphh 537/template sections and block display

## Description
This PR implements the display components for requirement section and requirement block for edit template page


## What type of PR is this? (check all applicable)

- [ X] 🍕 Feature

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-537



## Steps to QA


1. Navigate to permit templates index page
2. Locate any existing requirement template and click edit (There should be some due to seeds)
3. Verify if sections and blocks are displayed
![image](https://github.com/bcgov/HOUS-permit-portal/assets/32022335/60975fc9-39d9-443e-bbe6-9fb9c419da77)
